controllers/articles.js

```diff
@@ -27,6 +27,11 @@ const getAverageRating = (data) => {
   return Number(average.toFixed(2));
 };
 
+const generateDescription = (body) => {
+  const cleanText = body.replace(/<\/?[^>]+(>|$)/g, '');
+  return cleanText.substring(0, 100).replace(/&nbsp;/g, '');
```

Comment:
Great work @avpaul for removing tags in the description when generating it to be stored in the database. But can you work on checking if those characters you specified ends in the middle of a word so that the word will be taken as a full word? ex: when have `.................meaning`. don't store `.....meani`

Reply:
@niyobobo I think this can be a different fix because I also have a concern about the same area where we are getting the description.

Reply:
Fixed to 100 words then we complete the last word if it's not complete.